app/SharedWebComponents/Pages/VoiceChat.razor.cs

```diff
@@ -77,11 +79,24 @@ private void OnSendPrompt()
                                 Name = voice
                             };
                         }
-                        SpeechSynthesis.Speak(utterance, duration =>
+                        try
+                        {
+                            SpeechSynthesis.Speak(utterance, duration =>
+                            {
+                                _isReadingResponse = false;
+                                StateHasChanged();
+                            });
+                        }
+                        catch
                         {
+                            // TODO: try and not use the method above as it is not really useful
```

Comment:
Can you explain this to me in a bit more detail? I'm trying to understand the concerns, the `catch`, what "specific type" is referring to.

Reply:
Added a much better comment.